dev-docs/modules/bidResponseFilter.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page_v2
+page_type: module
+title: Module - Bid response filter
+description: Converts bid currency into ad server currency based on data in a supplied exchange rate file.
```

Comment:
@mkomorski I guess this is a copy and paste error ;)

Reply:
yup :) fixed